 <html>
	<head>
		<title>Bioinformatics</title>
    <style>
body {
  background-color: black;
  color: black;
  font-family: arial; 
  background-image: url(https://d2v9y0dukr6mq2.cloudfront.net/video/thumbnail/EFODv5zYgijxtshlo/videoblocks-science-technology-background-with-double-helix-strand_hhxcmbhgbz_thumbnail-full01.png);
  background-repeat: repeat;
         }
#bioinformatics {
 color: white;
 background-color: black;
}
#my-menu {
  font-family: calibri;
  font-weight: bold;
  font-size: 22px;
  background-color: black;
  color: white
}
#dna2 {
  padding: 0px 50px 0px 20px;
}
#dna3 {
  padding: 0px 50px 0px 20px;
}
#dna4 {
   padding: 0px 50px 0px 20px;
}
#dna5 {
   padding: 20px 100px 50px 20px;
}
#contents {
  background-color: black;
  color: white;
}
#introduction {
  background-color: black;
  color: white;
  font-weight: bold;
}
#history{
  background-color: black;
  color: white;
}
#sequences{
  background-color: black;
  color: white;
}
#goals{
  background-color: black;
  color: white;
}
#relation-to-other-fields{
  background-color: black;
  color: white;
}
#sequence-analysis {
  background-color: black;
  color: white;
  font-weight: bold;
}
#dna-sequencing {
  background-color: black;
  color: white;
}
#sequence-assembly {
  background-color: black;
  color: white;
}
#genome-annotation {
  background-color: black;
  color: white;
}
#computational-evolutionary-biology {
  background-color: black;
  color: white;
}
#comparative-genomics {
  background-color: black;
  color: white;
}
#pan-genomics {
  background-color: black;
  color: white;
}
#genetics-of-disease {
  background-color: black;
  color: white;
}
#analysis-of-mutation-in-cancer {
  background-color: black;
  color: white;
}
#gene-and-protein-expression {
  background-color: black;
  color: white;
  font-weight: bold;
}
#analysis-of-gene-expression {
  background-color: black;
  color: white;
}
#analysis-of-protein-expression {
background-color: black;
  color: white;
}
#analysis-of-regulation {
  background-color: black;
  color: white;
}
#analysis-of-cellular-organization {
  background-color: black;
  color: white;
  font-weight: bold;
}
#microscopy-and-image-analysis {
  background-color: black;
  color: white;
}
#protein-locatization {
  background-color: black;
  color: white;
}
#nuclear-organization-of-chromatin {
  background-color: black;
  color: white;
}
#structural-bioinformatics {
  background-color: black;
  color: white;
  font-weight: bold;
}
#network-and-systems-biology {
  background-color: black;
  color: white;
  font-weight: bold;
}
#molecular-interaction-networks {
  background-color: black;
  color: white;
}
#contact {
  background-color: black;
  color: white;
}
    </style>
    </head>
    

	<body>
    
    <img id="dna1"  align="center" width="1333px" alt="DNI"
src="https://previews.dropbox.com/p/thumb/AAuqQxU3UOvElHF6YgbWA6j3pmk2myD9-YNQMhnxDMuoLNhE3P2SqlVhV4ejijz0koEVd7YGLdFwIJUS06qh5R6CzG59UvM2u2Tclx88m9fCkoUFB76R9VaDvKh05XJfDP7a2skLzkCAm1PY-BnwGawSMX-q4C_JiVo7HS6Drx2Ilpz2Kp28GrLGqF4obqdKBb224bKhLw7hwEuCMJ0aiDIK-TCznHEg1N7RLjrljvedfFFDoEkQgh4xpscInez1gQ2NRBIdM4C9nlu-1KRKAy7-9W8rF3QuSp7Xi34SFBd35gy6dAwLHb1qxpQ_dWHp8hk7o_kSaeqbWtsGWN6zltjfWgID-d3yev7EPTayWysvGw/p.jpeg?fv_content=true&size_mode=5">
    
<div id="my-menu" align="right">
	<a href="#contents">Home</a> | 
	<a href="#contact">Contact</a> | 
	<a href="#introduction">About</a> |
	<a href="#map">Map</a>
</div>
        
    <h1 id="bioinformatics">Bioinformatics</h1> 
  
  <div>
    <img id="dna2"  align="right" width="550px" alt="DNI"
src="https://miro.medium.com/max/1024/0*a7jK9hhXZlHoah3l"><br>
  	
<p align="justify">Is an interdisciplinary field that develops methods and software tools for understanding biological data. As an interdisciplinary field of science, bioinformatics combines biology, computer science, information engineering, mathematics and statistics to analyze and interpret biological data. Bioinformatics has been used for in silico analyses of biological queries using mathematical and statistical techniques.</p>

<p align= "justify">Bioinformatics includes biological studies that use computer programming as part of their methodology, as well as a specific analysis "pipelines" that are repeatedly used, particularly in the field of genomics.Common uses of bioinformatics include the identification of candidates genes and single nucleotide polymorphisms (SNPs). Often, such identification is made with the aim of better understanding the genetic basis of disease, unique adaptations, desirable properties (esp. in agricultural species), or differences between populations. In a less formal way, bioinformatics also tries to understand the organisational principles within nucleic acid and protein sequences, called proteomics.</p><br><br>
   </div>
    
    <h2 id=contents>Contents</h2>
    <ul>
      <li><a href="#introduction">Introduction</a></li>
      <li><a href="#history">History</a></li>
      <li><a href="#sequences">Sequences</a></li>
      <li><a href="#goals">Goals</a></li>
      <li><a href="#relation-to-other-fields">Relation to other fields</a></li>
      <li><a href="#sequence-analysis">Sequence analysis</a></li>
      <li><a href="#dna-sequencing">DNA sequencing</a></li>
      <li><a href="#sequence-assembly">Sequence assembly</a></li>
      <li><a href="#genome-annotation">Genome annotation</a></li>
      <li><a href="#computational-evolutionary-biology">Computational evolutionary biology</a></li>
      <li><a href="#comparative-genomics">Comparative genomics</a></li>
      <li><a href="#pan-genomics">Pan genomics</a></li>
      <li><a href="#genetics-of-disease">Genetics of disease</a></li>
      <li><a href="#analysis-of-mutation-in-cancer">Analysis of mutations in cancer</li>
      <li><a href="#gene-and-protein-expression">Gene and protein expression</a></li>
      <li><a href="#analysis-of-gene-expression">Analysis of gene expression</a></li>
      <li><a href="#analysis-of-protein-expression">Analysis of protein expression</li>
      <li><a href="#analysis-of-regulation">Analysis of regulation</a></li>
      <li><a href="#analysis-of-cellular-organization">Analysis of cellular organization</a></li>
      <li><a href="#microscopy-and-image-analysis">Microscopy and image analysis</li>
      <li><a href="#protein-locatization">Protein localization</a></li>
      <li><a href="#nuclear-organization-of-chromatin">Nuclear organization of chromatin</li>
      <li><a href="#structural-bioinformatics">Structural bioinformatics</a></li>
      <li><a href="#network-and-systems-biology">Network and systems biology</a></li>
      <li><a href="#molecular-interaction-networks">Molecular interaction networks</a></li>
     </ul>
  
  <h1 id="introduction">INTRODUCTION</h1>
    
    <p align="justify">Bioinformatics has become an important part of many areas of biology. In experimental molecular biology, bioinformatics techniques such as image<br> and signal processing allow extraction of useful results from large amounts of raw data. In the field of genetics, it aids in sequencing and annotating<br> genomes and their observed mutations. It plays a role in the text mining of biological literature and the development of biological and gene ontologies<br> to organize and query biological data. It also plays a role in the analysis of gene and protein expression and regulation. Bioinformatics tools aid in<br> comparing, analyzing and interpreting of genetic and genomic data and more generally in the understanding of evolutionary aspects of molecular<br> biology. At a more integrative level, it helps analyze and catalogue the biological pathways and networks that are an important part of systems<br> biology. In structural biology, it aids in the simulation and modeling of DNA, RNA, proteins as well as biomolecular interactions.</p>
    
    <h2 id="history">History</h2>
    
    <p align="justify">Historically, the term bioinformatics did not mean what it means today.<br> Paulien Hogeweg and Ben Hesper coined it in 1970 to refer to the study of information processes in biotic systems.<br> This definition placed bioinformatics as a field parallel to biochemistry (the study of chemical processes in biological systems).</p>
    
    <h3 id="sequences">Sequences</h3>
    
    <img id="dna4"  align="left" width="390px" alt="Sequences of genetic material"
src="https://upload.wikimedia.org/wikipedia/commons/thumb/a/a0/Example_DNA_sequence.png/220px-Example_DNA_sequence.png">
    
    <p align="justify">Computers became essential in molecular biology when protein sequences became available after<br> Frederick Sanger determined the sequence of insulin in the early 1950s. Comparing multiple<br> sequences manually turned out to be impractical. A pioneer in the field was Margaret Oakley Dayhoff.<br> She compiled one of the first protein sequence databases, initially published as book and pioneered<br> methods of sequence alignment and molecular evolution. Another early contributor to bioinformatics<br> was Elvin A. Kabat, who pioneered biological sequence analysis in 1970 with his comprehensive<br> volumes of antibody sequences released with Tai Te Wu between 1980 and 1991. </p>
  
 
    
    <h2 id="goals">Goals</h2>
   <p align="justify">To study how normal cellular activities are altered in different disease states, the biological data must be combined to form a comprehensive picture<br> of these activities. Therefore, the field of bioinformatics has evolved such that the most pressing task now involves the analysis<br> and interpretation of various types of data. This includes nucleotide and amino acid sequences, protein domains, and protein structures.<br> The actual process of analyzing and interpreting data is referred to as computational biology. Important sub-disciplines within bioinformatics<br> and computational biology include:</p>
    <ul>
      <li>Development and implementation of computer programs that enable efficient access to, management and use of, various types<br> of information</li>
      <li>Development of new algorithms (mathematical formulas) and statistical measures that assess relationships among members of<br> large data sets. For example, there are methods to locate a gene within a sequence, to predict protein structure and/or function<br> and to cluster protein sequences into families of related sequences.</li>
    </ul>
    
    <p align="justify">The primary goal of bioinformatics is to increase the understanding of biological processes. What sets it apart from other approaches, however, is its focus on developing<br> and applying computationally intensive techniques to achieve this goal. Examples include: pattern recognition, data mining, machine learning algorithms, and visualization.<br> Major research efforts in the field include sequence alignment, gene finding, genome assembly, drug design, drug discovery, protein structure alignment, protein structure<br> prediction, prediction of gene expression and protein-interactions, genome-wide association studies, the modeling of evolution and cell division/mitosis.</p>

<p align="justify">Bioinformatics now entails the creation and advancement of databases, algorithms, computational and statistical techniques, and theory to solve formal and practical problems<br> arising from the management and analysis of biological data.</p>

<p align="justify">Over the past few decades, rapid developments in genomic and other molecular research technologies and developments in information technologies have combined to produce<br> a tremendous amount of information related to molecular biology. Bioinformatics is the name given to these mathematical and computing approaches used to glean<br> understanding of biological processes.</p>

<p align="justify">Common activities in bioinformatics include mapping and analyzing DNA and protein sequences, aligning DNA and protein sequences to compare them, and creating<br> and viewing 3-D models of protein structures.<p/>
    
    <h2 id="relation-to-other-fields">Relation to other fields</h2>
    
    <p align="justify">Bioinformatics is a science field that is similar to but distinct from biological computation,<br> while it is often considered synonymous to computational biology. Biological computation uses bioengineering and<br> biology to build biological computers, whereas bioinformatics uses computation to better understand biology.<br> Bioinformatics and computational biology involve the analysis of biological data, particularly DNA, RNA, and<br> protein sequences. The field of bioinformatics experienced explosive growth starting in the mid-1990s,<br> driven largely by the Human Genome Project and by rapid advances in DNA sequencing technology.<br>

Analyzing biological data to produce meaningful information involves writing and running software programs that<br> use algorithms from graph theory, artificial intelligence, soft computing, data mining,<br> image processing, and computer simulation. The algorithms in turn depend on theoretical foundations such as discrete<br> mathematics, control theory, system theory, information theory, and statistics.</p>
   
    <div>
    <h1 id="sequence-analysis">SEQUENCE ANALYSIS</h1>
    <img id="dna3"  align="left" width="500px" alt="DNI"
src="https://www.mayo.edu/-/media/kcms/gbs/research/images/2019/02/06/15/18/genetics-bioinformatics-shu-571040611-8col.jpg">
    
    <p align="justify">Since the Phage Φ-X174 was sequenced in 1977, the DNA sequences of thousands of organisms have<br> been decoded and stored in databases. This sequence information is analyzed to determine genes that<br> encode proteins, RNA genes, regulatory sequences, structural motifs, and repetitive sequences.<br> A comparison of genes within a species or between different species can show similarities between<br> protein functions, or relations between species (the use of molecular systematics to construct<br> phylogenetic trees). With the growing amount of data, it long ago became impractical to analyze DNA<br> sequences manually. Today, computer programs such as BLAST are used daily to search sequences<br> from more than 260 000 organisms, containing over 190 billion nucleotides. These programs<br> can compensate for mutations (exchanged, deleted or inserted bases) in the DNA sequence, to<br> identify sequences that are related, but not identical. A variant of this sequence alignment is used in the<br> sequencing process itself. For the special task of taxonomic classification of sequence snippets, modern<br> k-mer based software like Kraken achieves throughput unreachable by alignment methods.</p>
    </div>
    
    <h2 id="dna-sequencing">DNA sequencing</h2>
    
    <p align="justify">Before sequences can be analyzed they have to be obtained from the data storage bank<br> example the Genbank. DNA sequencing is still a non-trivial problem as the raw data may be noisy or<br> afflicted by weak signals. Algorithms have been developed for base calling for the various experimental approaches to DNA<br> sequencing.<p/>
    
    <h2 id="sequence-assembly">Sequence assembly</h2>
    
    <p align="justify">Most DNA sequencing techniques produce short fragments of sequence that need to be<br> assembled to obtain complete gene or genome sequences. The so-called shotgun sequencing technique<br> (which was used, for example, by The Institute for Genomic Research (TIGR) to sequence the first bacterial genome,<br> Haemophilus influenzae) generates the sequences of many thousands of small DNA fragments<br> (ranging from 35 to 900 nucleotides long, depending on the sequencing technology). The ends of these fragments overlap and,<br> when aligned properly by a genome assembly program, can be used to reconstruct the complete genome.<br> Shotgun sequencing yields sequence data quickly, but the task of assembling the fragments can be quite<br> complicated for larger genomes. For a genome as large as the human genome, it may take many days of CPU<br> time on large-memory, multiprocessor computers to assemble the fragments, and the resulting assembly usually contains<br> numerous gaps that must be filled in later. Shotgun sequencing is the method of choice for virtually all<br> genomes sequenced today[when?], and genome assembly algorithms are a critical area of bioinformatics research.</p>
    
    <h2 id="genome-annotation">Genome annotation</h2>
    
    <p align="justify">In the context of genomics, annotation is the process of marking the genes and<br> other biological features in a DNA sequence. This process needs to be automated because most genomes are<br> too large to annotate by hand, not to mention the desire to annotate as many genomes as possible, as the rate of<br> sequencing has ceased to pose a bottleneck. Annotation is made possible by the fact that genes have recognisable<br> start and stop regions, although the exact sequence found in these regions can vary between genes.</p>

<p align="justify">The first description of a comprehensive genome annotation system was published in 1995<br> by the team at The Institute for Genomic Research that performed the first complete sequencing and analysis of<br> the genome of a free-living organism, the bacterium Haemophilus influenzae. Owen White designed and<br> built a software system to identify the genes encoding all proteins, transfer RNAs, ribosomal RNAs (and other sites)<br> and to make initial functional assignments. Most current genome annotation systems work similarly, but the<br> programs available for analysis of genomic DNA, such as the GeneMark program trained and used to find protein-coding<br> genes in Haemophilus influenzae, are constantly changing and improving.</p>

    </p>Following the goals that the Human Genome Project left to achieve after its closure in 2003,<br> a new project developed by the National Human Genome Research Institute in the U.S appeared.<br>The so-called ENCODE project is a collaborative data collection of the functional elements of the human genome<br> that uses next-generation DNA-sequencing technologies and genomic tiling arrays, technologies able to<br> automatically generate large amounts of data at a dramatically reduced per-base cost but with the same accuracy<br> (base call error) and fidelity (assembly error).</p>

<h2 id="computational-evolutionary-biology">Computational evolutionary biology</h2>

<p align="justify">Evolutionary biology is the study of the origin and descent of species, as well as their change over time.<br> Informatics has assisted evolutionary biologists by enabling researchers to</p>

<ul>
  <li>trace the evolution of a large number of organisms by measuring changes in their DNA, rather than through physical taxonomy or physiological observations alone,</li>
  <li>compare entire genomes, which permits the study of more complex evolutionary events, such as gene duplication, horizontal gene transfer, and the prediction of factors<br> important in bacterial speciation,</li>
  <li>build complex computational population genetics models to predict the outcome of the system over time</li>
  <li>track and share information on an increasingly large number of species and organisms</li>
</ul>
  
  <p align="justify">Future work endeavours to reconstruct the now more complex tree of life.</p>
  <p align="justify">The area of research within computer science that uses genetic algorithms is sometimes confused<br> with computational evolutionary biology, but the two areas are not necessarily related.</p>
    
  <h2 id="comparative-genomics">Comparative genomics</h2>
  
  <p align="justify">The core of comparative genome analysis is the establishment of the correspondence<br> between genes (orthology analysis) or other genomic features in different organisms. It is these intergenomic<br>maps that make it possible to trace the evolutionary processes responsible for the divergence of two genomes.<br> A multitude of evolutionary events acting at various organizational levels shape genome evolution.<br> At the lowest level, point mutations affect individual nucleotides. At a higher level, large chromosomal segments undergo<br> duplication, lateral transfer, inversion, transposition, deletion and insertion.<br> Ultimately, whole genomes are involved in processes of hybridization, polyploidization and endosymbiosis, often leading to rapid speciation.<br> The complexity of genome evolution poses many exciting challenges to developers of mathematical models and<br> algorithms, who have recourse to a spectrum of algorithmic, statistical and mathematical <br>techniques, ranging from exact, heuristics, fixed parameter and approximation algorithms for problems based on parsimony<br> models to Markov chain Monte Carlo algorithms for Bayesian analysis of problems based on probabilistic models.<br>

Many of these studies are based on the detection of sequence homology to assign sequences to protein families.</p>
  
  <h2 id="pan-genomics">Pan genomics</h2>
  
  <p align="justify">Pan genomics is a concept introduced in 2005 by Tettelin and Medini which eventually<br> took root in bioinformatics. Pan genome is the complete gene repertoire of a particular taxonomic<br> group: although initially applied to closely related strains of a species, it can be applied to a larger<br> context like genus, phylum etc. It is divided in two parts- The Core genome: Set of genes common to all the genomes<br> under study (These are often housekeeping genes vital for survival) and The Dispensable/Flexible Genome: Set of genes<br> not present in all but one or some genomes under study. A bioinformatics tool BPGA can be used to<br> characterize the Pan Genome of bacterial species.</p>
  
  <h2 id="genetics-of-disease">Genetics of disease</h2>
  
  <p align="justify">With the advent of next-generation sequencing we are obtaining enough sequence<br> data to map the genes of complex diseases infertility, breast cancer or Alzheimer's disease.<br> Genome-wide association studies are a useful approach to pinpoint the mutations responsible for such complex diseases.<br> Through these studies, thousands of DNA variants have been identified that are associated with similar diseases and traits.<br> Furthermore, the possibility for genes to be used at prognosis, diagnosis or treatment is one of the most essential applications. Many studies are discussing both the promising ways to choose the genes to<br> be used and the problems and pitfalls of using genes to predict disease presence or prognosis.</p>
    
  <h2 id="analysis-of-mutation-in-cancer">Analysis of mutations in cancer</h2>
  
  <img id="dna4"  align="right" width="600px" alt="DNI"
src="https://webcdn.leica-microsystems.com/fileadmin/academy/user_upload/dna-analysis-wf.jpg">
  
  <p align="justify">In cancer, the genomes of affected cells are rearranged in complex or even unpredictable<br> ways.Massive sequencing efforts are used to identify previously unknown point mutations<br> in a variety of genes in cancer. Bioinformaticians continue to produce specialized automated<br> systems to manage the sheer volume of sequence data produced, and they create new<br> algorithms and software to compare the sequencing results to the growing collection<br> of human genome sequences and germline polymorphisms. New physical detection<br> technologies are employed, such as oligonucleotide microarrays to identify chromosomal<br> gains and losses (called comparative genomic hybridization), and single-nucleotide<br> polymorphism arrays to detect known point mutations. These detection methods<br> simultaneously measure several hundred thousand sites throughout the genome, and when<br> used in high-throughput to measure thousands of samples, generate terabytes of data<br> per experiment. Again the massive amounts and new types of data generate new<br> opportunities for bioinformaticians The data is often found to contain considerable variability<br>, or noise, and thus Hidden Markov model and change-point analysis methods are<br> being developed to infer real copy number changes.</p>

<p align="justify">Two important principles can be used in the analysis of cancer genomes bioinformatically<br> pertaining to the identification of mutations in the exome. First, cancer is a disease<br> of accumulated somatic mutations in genes. Second cancer contains driver mutations which<br> need to be distinguished from passengers.</p>

<p align="justify">With the breakthroughs that this next-generation sequencing technology is providing to the<br> field of Bioinformatics, cancer genomics could drastically change. These new methods and<br> software allow bioinformaticians to sequence many cancer genomes quickly and affordably. This could create a more flexible process for classifying types of cancer by analysis of<br> cancer driven mutations in the genome. Furthermore, tracking of patients while the disease progresses may be possible in the future with the sequence of cancer samples.</p>

<p>Another type of data that requires novel informatics development is the analysis of lesions found to be recurrent among many tumors.</p>
  
  <h1 id="gene-and-protein-expression">GENE AND PROTEIN EXPRESSION<h1>
    
    <h2 id="analysis-of-gene-expression">Analysis of gene expression</h2>
    
    <p align="justify">The expression of many genes can be determined by measuring mRNA levels with multiple<br> techniques including microarrays, expressed cDNA sequence tag (EST) sequencing,<br> serial analysis of gene expression (SAGE) tag sequencing, massively parallel signature sequencing (MPSS), RNA-Seq, also known as "Whole Transcriptome Shotgun Sequencing" (WTSS), or various applications of multiplexed<br> in-situ hybridization. All of these techniques are extremely noise-prone and/or subject to bias in the biological<br> measurement, and a major research area in computational biology involves developing statistical tools to separate signal from<br> noise in high-throughput gene expression studies. Such studies are often used to determine<br> the genes implicated in a disorder: one might compare microarray data from cancerous epithelial cells to data from non-cancerous<br> cells to determine the transcripts that are up-regulated and down-regulated in a particular<br> population of cancer cells.</p>
    
    <h2 id="analysis-of-protein-expression">Analysis of protein expression</h2>
    
    <p align="justify">Protein microarrays and high throughput (HT) mass spectrometry (MS)<br> can provide a snapshot of the proteins present in a biological sample. Bioinformatics is very much<br> involved in making sense of protein microarray and HT MS data; the former approach faces similar problems as with microarrays<br> targeted at mRNA, the latter involves the problem of matching large amounts of mass data against predicted<br> masses from protein sequence databases, and the complicated statistical analysis of samples where<br> multiple, but incomplete peptides from each protein are detected. Cellular protein localization in a tissue context<br> can be achieved through affinity proteomics displayed as spatial data based on immunohistochemistry and tissue microarrays.</p>
    
    <h2 id="analysis-of-regulation">Analysis of regulation</h2>
    
    <p align="justify">Gene regulation is the complex orchestration of events by which a signal, potentially an extracellular signal such as a hormone, eventually leads to an<br> increase or decrease in the activity of one or more proteins. Bioinformatics techniques have been applied to explore various steps in this process.<br>

For example, gene expression can be regulated by nearby elements in the genome. Promoter analysis involves the identification and study of<br> sequence motifs in the DNA surrounding the coding region of a gene. These motifs influence the extent to which that region is transcribed into mRNA.<br> Enhancer elements far away from the promoter can also regulate gene expression, through three-dimensional looping interactions. These interactions<br> can be determined by bioinformatic analysis of chromosome conformation capture experiments.

Expression data can be used to infer gene regulation:<br> one might compare microarray data from a wide variety of states of an organism to form hypotheses about the genes involved in each state.<br> In a single-cell organism, one might compare stages of the cell cycle, along with various stress conditions (heat shock, starvation, etc.)<br> One can then apply clustering algorithms to that expression data to determine which genes are co-expressed. For example, the upstream<br> regions (promoters) of co-expressed genes can be searched for over-represented regulatory elements. Examples of clustering algorithms applied<br> in gene clustering are k-means clustering, self-organizing maps (SOMs), hierarchical clustering, and consensus clustering methods.</p>
    
   
    <h1 id="analysis-of-cellular-organization">ANALYSIS OF CELLULAR ORGANIZATION</h1>
    <p align="justify">Several approaches have been developed to analyze the location of organelles,<br> genes, proteins, and other components within cells.<br> This is relevant as the location of these components affects the events within a cell and thus helps us to predict<br> the behavior of biological systems. A gene ontology category, cellular compartment, has been devised to capture<br> subcellular localization in many biological databases.</p>
   
    <h2 id="microscopy-and-image-analysis">Microscopy and image analysis</h2>
    
    <p>Microscopic pictures allow us to locate both organelles as well as molecules. It may also help us to<br> distinguish between normal and abnormal cells, e.g. in cancer.</p>
    
    <h2 id="protein-locatization">Protein localization</h2>
    
    <p align="justify">The localization of proteins helps us to evaluate the role of a protein.<br> For instance, if a protein is found in the nucleus it may be involved in gene regulation or splicing.<br> By contrast, if a protein is found in mitochondria, it may be involved in respiration or other metabolic processes.<br> Protein localization is thus an important component of protein function prediction.<br> There are well developed protein subcellular localization prediction resources available, including protein subcellular location databases,<br> and prediction tools.</p>
    
    <h2 id="nuclear-organization-of-chromatin">Nuclear organization of chromatin</h2>
    
    <p align="justify">Data from high-throughput chromosome conformation capture experiments,<br> such as Hi-C (experiment) and ChIA-PET, can provide information on the spatial proximity of DNA loci.<br> Analysis of these experiments can determine the three-dimensional structure and nuclear organization of chromatin.<br> Bioinformatic challenges in this field include partitioning the genome into domains,<br> such as Topologically Associating Domains (TADs), that are organised together in three-dimensional space.</p>
    
    <h1 id="structural-bioinformatics">STRUCTURAL BIOINFORMATICS</h1>
    
    <img id="dna4"  align="right" width="600px" alt="DNI"
src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/50/1kqf_opm.png/504px-1kqf_opm.png">
    
    <p align="justify">Protein structure prediction is another important application of bioinformatics.<br> The amino acid sequence of a protein, the so-called primary structure, can be<br> easily determined from the sequence on the gene that codes for it. In the vast<br> majority of cases, this primary structure uniquely determines a structure in its<br> native environment. (Of course, there are exceptions, such as the bovine<br> spongiform encephalopathy (mad cow disease) prion.) Knowledge of this<br> structure is vital in understanding the function of the protein. Structural<br> information is usually classified as one of secondary, tertiary and quaternary<br> structure. A viable general solution to such predictions remains an open problem.<br> Most efforts have so far been directed towards heuristics that work<br> most of the time.</p>
    
    <p align="justify">One of the key ideas in bioinformatics is the notion of homology. In the<br> genomic branch of bioinformatics, homology is used to predict the function of<br> a gene: if the sequence of gene A, whose function is known, is homologous to<br> the sequence of gene B, whose function is unknown, one could infer that B<br> may share A's function. In the structural branch of bioinformatics, homology is<br> used to determine which parts of a protein are important in structure formation<br> and interaction with other proteins. In a technique called homology modeling,<br> this information is used to predict the structure of a protein once the structure<br> of a homologous protein is known. This currently remains the only way to<br> predict protein structures reliably.</p>
    
    <p align="justify">One example of this is hemoglobin in humans and the hemoglobin in legumes<br> (leghemoglobin), which are distant relatives from the same protein superfamily.<br> Both serve the same purpose of transporting oxygen in the organism.<br> Although both of these proteins have completely different<br> amino acid sequences, their protein structures are virtually identical<br> which reflects their<br> near identical purposes and shared ancestor</p>
    
    <p align="justify">Other techniques for predicting protein structure include protein<br> threading and de novo (from scratch) physics-based modeling</p>
    
    <p align="justify">Another aspect of structural bioinformatics include the use of protein structures<br> for Virtual Screening models such as Quantitative Structure-Activity Relationship<br> models and proteochemometric models (PCM). Furthermore, a protein's crystal<br> structure can be used in simulation of for example ligand-binding studies and<br> in silico mutagenesis studies.</p>
    
    <h1 id="network-and-systems-biology">NETWORK AND SYSTEMS BIOLOGY</h1>
    
    <p align="justify">Network analysis seeks to understand the relationships within biological<br> networks such as metabolic or protein–protein interaction networks. Although biological networks can be constructed from<br> a single type of molecule or entity (such as genes), network biology often attempts to integrate many<br> different data types, such as proteins, small molecules, gene expression data, and others, which are all connected<br> physically, functionally, or both.

Systems biology involves the use of computer simulations of cellular subsystems (such as the networks<br> of metabolites and enzymes that comprise metabolism, signal transduction pathways and gene regulatory<br> networks) to both analyze and visualize the complex connections of these cellular processes. Artificial life or<br> virtual evolution attempts to understand evolutionary processes via the computer simulation of simple<br> (artificial) life forms.
      
    <h2 id="molecular-interaction-networks">Molecular interaction networks</h2>
    
    <img id="dna4"  align="left" width="150px" alt="Sequences of genetic material"
src="https://upload.wikimedia.org/wikipedia/commons/thumb/b/b4/The_protein_interaction_network_of_Treponema_pallidum.png/200px-The_protein_interaction_network_of_Treponema_pallidum.png">
    
    <p align="justify">Tens of thousands of three-dimensional protein structures have been determined<br> by X-ray crystallography and protein nuclear magnetic resonance spectroscopy (protein NMR) and a central<br> question in structural bioinformatics is whether it is practical to predict possible protein–protein interactions<br> only based on these 3D shapes, without performing protein–protein interaction experiments.<br> A variety of methods have been developed to tackle the protein–protein docking problem, though it seems that there is still<br> much work to be done in this field.

Other interactions encountered in the field include Protein–ligand (including drug) and protein–peptide.<br> Molecular dynamic simulation of movement of atoms about rotatable bonds is the fundamental<br> principle behind computational algorithms, termed docking algorithms, for studying molecular interactions.</p>
    
   <a href="https://en.wikipedia.org/wiki/Bioinformatics">Information by wikipedia</a>
    
    <h2 id="contact">Contact</h2>
    <p align="justify">Name: Thiago Luiz Pereira<br>
       Telephone: +34 6518449XX<br>
      e-mail: t.luiz...@gmail.com</p>
    </>
    
    <iframe id="map" src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2992.17240602541!2d2.1747513147542303!3d41.41377600232303!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x12a4a2d1a81f4677%3A0xf637866434aa2bd!2sBioinformatics%20Barcelona%20Association%20(BIB)!5e0!3m2!1sen!2ses!4v1582126218282!5m2!1sen!2ses" width="1333px" height="300" frameborder="0" style="border:0;" allowfullscreen=""></iframe>
	</body>
</html>
								